go.mod

```diff
@@ -3,21 +3,26 @@ module github.com/nmstate/kubernetes-nmstate
 go 1.13.7
 
 require (
+	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/aktau/github-release v0.7.2
 	github.com/evanphx/json-patch v4.5.0+incompatible
 	github.com/go-logr/logr v0.1.0
 	github.com/go-openapi/spec v0.19.4
 	github.com/gobwas/glob v0.2.3
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/onsi/ginkgo v1.10.1
-	github.com/onsi/gomega v1.7.0
+	github.com/onsi/ginkgo v1.10.3
+	github.com/onsi/gomega v1.7.1
```

Comment:
This is unrelated to the test, can you justify why do we need to bump ginkgo ?, it add a lot of noise to the review, we should do bumping at different PR, to check results and easy review.